Components/card/card.html

```diff
@@ -0,0 +1,458 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Card | Blaze UI</title>
+
+    <link href="/css/shared.css" rel="stylesheet">
+    <link href="/css/documentation.css" rel="stylesheet">
```

Comment:
Can u please merge this all css file into one and then add only single link into HTML.